quickblue/src/Explorer/Models.hs

```diff
@@ -74,6 +76,36 @@ data Items a = Items
   , total :: Int
   } deriving (Show, Generic, FromJSON)
 
+data FullTx = FullTx
+  { blockHash  :: BlockHash
+  , blockIndex :: Int
+  , hash       :: TxHash
+  , inputs     :: [FullTxIn]
+  , outputs    :: [FullTxOut]
+  } deriving (Show, Generic, FromJSON)
+
+instance ToCardanoTx FullTx Tx.SettledTx where
+  toCardanoTx FullTx{..} =
+    Tx.SettledTx
+      { blockHash = Tx.BlockHash $ unBlockHash blockHash
+      , txIndex   = blockIndex
+      , hash      = Tx.TxHash $ unTxHash hash
+      , inputs    = fmap toCardanoTx inputs
+      , outputs   = fmap toCardanoTx outputs
+      } 
+
+data FullTxIn = FullTxIn
+  { out :: FullTxOut
+  } deriving (Show, Generic, FromJSON)
+
+instance ToCardanoTx FullTxIn Tx.FullTxIn where
+  toCardanoTx FullTxIn{..} =
+    Tx.FullTxIn
+      { fullTxInTxOut = toCardanoTx out
+      -- actually, we don't need this field att all, so we keep it default
+      , fullTxInType  = Tx.ConsumeSimpleScriptAddress
```

Comment:
why only ConsumeSimpleScriptAddress?

Reply:
We do not need actual redeemer in our models (cause it contains just indexes of outs in tx)

Reply:
We do not need this field at all, but model that already exists  contains it, so i put the simplest value by default